dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -147,26 +148,32 @@ class SentryStackTraceFactory {
     final scheme = frame.uri.scheme;
 
     if (scheme.isEmpty) {
-      return true;
+      // Early bail out.
+      return _options.isStackFrameInAppDefault;
     }
+    // The following code depends on the scheme being set.
 
-    for (final include in _options.inAppIncludes) {
-      if (frame.package != null && frame.package == include) {
+    final package = frame.package;
+    if (package != null) {
+      if (_options.inAppIncludes.contains(package)) {
         return true;
       }
-    }
 
-    for (final exclude in _options.inAppExcludes) {
-      if (frame.package != null && frame.package == exclude) {
+      if (_options.inAppExcludes.contains(package)) {
         return false;
       }
     }
 
-    if (frame.isCore ||
-        (frame.uri.scheme == 'package' && frame.package == 'flutter')) {
+    if (frame.isCore) {
```

Comment:
why don't we check `frame.uri.scheme == 'package'` anymore`? isn't needed anymore?

Reply:
The `frame.uri.scheme == 'package'` check is redundant. If the scheme isn't package, `frame.package` would be null.